pkgs/by-name/wl/wl-gammarelay-applet/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+  wayland,
+  libxkbcommon,
+  fontconfig,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wl-gammarelay-applet";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "lgbishop";
+    repo = "wl-gammarelay-applet";
+    rev = "8a0d9e6364d7445fc69c59b2f168cfec91c2fe87";
+    sha256 = "sha256-t6bycmaquZ0IMs/WnAzkz5FnIWKIq0BTbeeoUFLeuYg=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+  buildInputs =
+    [
+    ]
+    ++ lib.optionals stdenv.isLinux [
+
+      fontconfig
+    ]
+    ++
+      lib.optionals stdenv.isDarwin
+        [
+        ];
+
+  postFixup = lib.optionalString stdenv.isLinux ''
+    patchelf $out/bin/wl-gammarelay-applet \
+      --add-rpath ${
+        lib.makeLibraryPath [
+          wayland
+          libxkbcommon
+          fontconfig
+        ]
+      }
+  '';
```

```suggestion
  runtimeDependencies = [
    wayland
    libxkbcommon
    fontconfig
  ];
```

Add `autoPatchelfHook` to `nativeBuildInputs`

---

Done: [https://github.com/lgbishop/nixpkgs/commit/a7f456d07e52b6b54fe655e0f6e003e2afe57331](url)